dotnet/src/Skills/Skills.OpenAPI/Extensions/KernelOpenApiExtensions.cs

```diff
@@ -49,6 +50,9 @@ public static class KernelOpenApiExtensions
         AuthenticateRequestAsyncCallback? authCallback = null,
         string? userAgent = "Microsoft-Semantic-Kernel",
         HttpRetryConfig? retryConfiguration = null,
+#pragma warning disable CA1054 // URI-like parameters should not be strings
+        string? serverUrlOverride = null,
```

Comment:
Let's get rid of the pragma and use `Uri` type. It's done elsewhere. If there is a reason it can't be done, let's add that reasoning here.